frontend/lib/src/components/elements/LinkButton/styled-components.ts

```diff
@@ -140,17 +141,10 @@ export const StyledSecondaryLinkButton = styled(
     color: theme.colors.bodyText,
   },
   "&:hover": {
-    borderColor: theme.colors.primary,
-    color: theme.colors.primary,
+    backgroundColor: theme.colors.darkenedBgMix15,
   },
   "&:active": {
-    color: theme.colors.white,
-    borderColor: theme.colors.primary,
-    backgroundColor: theme.colors.primary,
-  },
-  "&:focus:not(:active)": {
-    borderColor: theme.colors.primary,
-    color: theme.colors.primary,
```

Comment:
nit: I think this was used to show the button via the primary color if it was selected via tab? Is it intended that we don't apply a different look other than the border shadow?